addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -970,8 +967,10 @@ registerWebsitePreviewTour("website_form_editable_content", {
         content: "Check that the new text value was correctly set",
         trigger: ":iframe section.s_website_form h3:contains(/^ABC$/)",
     },
-    {   content: "Remove the dropped column",
-        trigger: ":iframe .oe_overlay.oe_active .oe_snippet_remove",
+    {
+        content: "Remove the dropped column",
+        //To remove :not(:visible) of the trigger, we should scrollTo top of body.
+        trigger: ":iframe .oe_overlay.oe_active .oe_snippet_remove:not(:visible)",
```

Comment:
I understood your commit message's point but I don't get this one. Can't this be fixed the same way as other by first hovering the snippet ?

All in all, I guess you know it but relying on the presence of `:visible` in the trigger string seems a bit weird. If the sole use case if something that has no with or height, surely the tour is indeed wrong in the first place as you can't really expect users to interact with those and thus the tour should be changed somehow (by hovering first like here?)

Reply:
I actually tried to do the hover and click in this case, but it doesn't work. I'll add a TODO to do it later and not block this PR.

Concerning the use of :visible or rather :not(:visible) in the tours, it is indeed not "logical" that we find ourselves with invisible triggers. However, in some cases (as in enterprise/test_l10n_be_hr_payroll_account/static/tests/tours/hr_contract_salary_tour.js ), this feature is useful.